mavlink_vehicles/mavlink_vehicles.hh

```diff
@@ -43,29 +44,59 @@ struct attitude : state_variable {
 };
 
 struct global_pos_int : state_variable {
-    int32_t lat;
-    int32_t lon;
-    int32_t alt;
+    int32_t lat; // Degrees * 1e7
+    int32_t lon; // Degrees * 1e7
+    int32_t alt; // Millimeters (AMSL)
+    global_pos_int() {}
+    global_pos_int(int32_t _lat, int32_t _lon, int32_t _alt)
```

Comment:
Unfortunately this is not possible because global_pos_int is inheriting from state_variable.